test/app/app.rb

```diff
@@ -26,7 +26,7 @@
 org = Organization.create!(name: 'test')
 org.users.create!(email: 'squirrel@example.com')
 
-user = User.find_by(email: 'squirrel@example.com')
+user = User.find_by(email: 'squirrel@example.com') # bumpity
```

Comment:
Table Scan: The database reads 100% (5000) of the of the rows in **users**, skipping any indexes.